manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Rainmeter.Rainmeter.Beta
+PackageVersion: 4.4.0.3506
+PackageLocale: en-US
+Publisher: Rainmeter
+PublisherUrl: https://www.rainmeter.net
+PublisherSupportUrl: https://forum.rainmeter.net
+#PrivacyUrl: 
+Author: Rainmeter Team
+PackageName: Rainmeter.Beta
```

Comment:
```suggestion
PackageName: Rainmeter (Beta)
```

Reply:
Isn't using the `.Beta` naming conventions recommended?
Google does it for [Chrome](https://github.com/microsoft/winget-pkgs/blob/master/manifests/g/Google/Chrome/Beta/92.0.4515.81/Google.Chrome.Beta.installer.yaml#L2).

Reply:
Not in the PackageName
https://github.com/microsoft/winget-pkgs/blob/master/manifests/g/Google/Chrome/Beta/92.0.4515.81/Google.Chrome.Beta.locale.en-US.yaml#L10

Reply:
Ah, I see. Thanks!